packages/client/components/AnalyticsPage.tsx

```diff
@@ -61,8 +62,26 @@ const AnalyticsPage = () => {
       window.analytics = mockSnippet
     }
   }, [])
-  const [isSegmentLoaded] = useScript(`https://cdn.segment.com/analytics.js/v1/${key}/analytics.min.js`, {crossOrigin: true})
+  const [isSegmentLoaded] = useScript(
+    `https://cdn.segment.com/analytics.js/v1/${key}/analytics.min.js`,
+    {
+      crossOrigin: true
+    }
+  )
   const atmosphere = useAtmosphere()
+  useEffect(() => {
+    if (window.__ACTION__.logRocketEnabled) {
+      const email = window.localStorage.getItem(LocalStorageKey.EMAIL)
+      LogRocket.init(window.__ACTION__.logRocketId, {
```

Comment:
it's possible that `logRocketEnabled` is true but `logRocketId` is null.
let's simplify by doing what we do with segment & sentry-- just send an optional key to the client. if the key is present, it's enabled, else disabled